scripts/count_tokens.py

```
@@ -31,11 +31,86 @@ def count_tokens_in_file(filepath, encoding):
+        print("| Extension | Tokens | Lines | Files | % Tokens |")
+        print("|-----------|--------|-------|-------|----------|")
+        for ext in sorted(tokens_by_extension.keys(), key=lambda x: tokens_by_extension[x], reverse=True):
+            tokens = tokens_by_extension[ext]
+            lines = lines_by_extension[ext]
+            files = files_by_extension[ext]
+            pct = (tokens / total_tokens * 100) if total_tokens > 0 else 0
+            print(f"| `{ext}` | {format_number(tokens)} | {format_number(lines)} | {files} | {pct:.1f}% |")
+    print()
+
+
+def print_text_results(base_path, total_tokens, total_lines, file_count,
```

Ruff (0.14.11)

Unused function argument: `base_path`

(ARG001)